src/blk/kernel/KernelDevice.cc

```diff
@@ -532,7 +532,7 @@ void KernelDevice::_aio_stop()
   }
 }
 
-void KernelDevice::_discard_update_threads()
+void KernelDevice::_discard_update_threads(bool discard_stop)
```

Comment:
This arg is no longer in use after changes suggested by Igor

Reply:
It is still in use. We partly reverted the initial modification which had eliminated discard_stop check in this function. So it's back there for now:
 if (!cct->_conf.get_val<bool>("bdev_enable_discard") || 
     !support_discard || discard_stop) {
    newcount = 0;
  }

Reply:
Ah, understood; used here and not in the class anymore. Thanks!